app/src/main/java/com/example/momolabfe/ui/record/RecordEditFragment.kt

```
@@ -206,63 +204,107 @@ class RecordEditFragment : Fragment() {
-            val no = item.exchangeNo
+        for (i in 0 until childCount) {
+            val child = container.getChildAt(i)
+            val itemBinding = ItemExchangeEditBinding.bind(child)
+
+            val no = currentExchanges.getOrNull(i)?.exchangeNo ?: (i + 1)
+
+            // 교환 시각
+            val timeStr = itemBinding.exchangeTimeValueEt.text.toString().trim()
+            val exchangeTime = try {
+                LocalTime.parse(timeStr, TIME_FORMATTER)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)